### DISCUSSION TITLE
jellyfin 10.9.7

### DISCUSSION THREAD

**Discussion by webodev-sean** [hrfee/jfa-go#348] [Q&A]
does this work on latest jellyfin?
carn't seem to connect to server what am i doing wrong

**Comment by hrfee** [comment]
I'm not sure why it should fail, the program still connects to Jellyfin for me on 10.9.8. Any specific error?

**Comment by webodev-sean** [comment]
I got it working now only problem I face now is app/config if I close
docker or reboot computer it will forget and how to port forward I want it
outside my network but not having any luck

**Reply by hrfee** [reply]
Neither seem like problems specific to this project, but for the first one; are you sure the configuration directory is mounted to the docker container? i.e. you included something like `-v /opt/my-jfa-go-config:/data` when creating the container. If you didn't, the config may not be permanent (although a reboot shouldn't cause it to go away).
